.github/workflows/kit.yml

```diff
@@ -0,0 +1,67 @@
+# ------------------------------------------------------------
+# Copyright (c) Microsoft Corporation and Dapr Contributors.
+# Licensed under the MIT License.
+# ------------------------------------------------------------
+
+name: kit
+
+on:
+  push:
+    branches:
+      - master
```

Comment:
`master` branch is not called `main` - this probably explains why PR action did not get triggered.